packit/upstream.py

```diff
@@ -384,14 +384,21 @@ def create_archive(self, version: str = None) -> str:
             outputs = self.get_output_from_action(
                 action=ActionName.create_archive, env=env
             )
-            if not outputs:
-                raise PackitException("No output from create-archive action.")
+            if self.package_config.archive_name:
+                archive_path = Path(
```

Comment:
hm, so we ignore the output? shouldn't the prio be "output" > "package_config.archive_name"?